packer/README.md

```diff
@@ -34,6 +34,16 @@ vagrant box remove centos7-katello-devel-stable
 sudo virsh vol-delete --pool default centos7-katello-devel-stable_vagrant_box_image_0.img
 ```
 
+### Creating version specific dev boxes
+
+To create a version specific dev box, simply run:
+
+```
+export VERSION=3.18
+packer build --var  packer_hostname=centos7-katello-$VERSION-stable.example.com --var ansible_variables=\'$(./scripts/versions_to_json.rb $VERSION)\'   centos7-katello-devel-stable.json
+```
+
+Using the above instructions, add the box to your vagrant, spin it up and verify everything looks sane.  Push to Vagrant cloud.
```

Comment:
I think we'll have to expand on the "push to vagrant cloud", but I could see that documented in the release documentation or toolbelt.

Reply:
yeah i'm happy to expand on that, let me get the credentials and run through the process and i can elaborate more

Reply:
Sounds good, here is what I use https://github.com/johnpmitsch/vagrant_cloud_scripts/blob/master/upload_to_vagrant_cloud feel free to take anything from there